internal/server/handler_v2.go

```diff
@@ -16,22 +16,64 @@
 package server
 
 import (
+	"bytes"
 	"context"
+	"io"
+	"net/http"
 
+	"github.com/datacommonsorg/mixer/internal/merger"
+	"github.com/datacommonsorg/mixer/internal/server/resource"
 	v1e "github.com/datacommonsorg/mixer/internal/server/v1/event"
 	v2 "github.com/datacommonsorg/mixer/internal/server/v2"
 	v2e "github.com/datacommonsorg/mixer/internal/server/v2/event"
-	v2observation "github.com/datacommonsorg/mixer/internal/server/v2/observation"
 	v2p "github.com/datacommonsorg/mixer/internal/server/v2/properties"
 	v2pv "github.com/datacommonsorg/mixer/internal/server/v2/propertyvalues"
 	"github.com/datacommonsorg/mixer/internal/server/v2/resolve"
 	"github.com/datacommonsorg/mixer/internal/util"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/reflect/protoreflect"
 
 	pbv2 "github.com/datacommonsorg/mixer/internal/proto/v2"
 )
 
+func fetchRemote(
+	metadata *resource.Metadata,
+	apiPath string,
+	in protoreflect.ProtoMessage,
+	out protoreflect.ProtoMessage,
```

Comment:
Use: proto.Message

https://godoc.corp.google.com/pkg/google3/third_party/golang/protobuf/v2/reflect/protoreflect/protoreflect#ProtoMessage

"This is declared in the protoreflect package to avoid a cyclic dependency; use the proto.Message type instead, which aliases this type."

Reply:
Nice!